### PR TITLE
rss2email: init at 3.9

### PR DESCRIPTION
**Pull request by jb55**
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/rss2email-sendmail-default.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/rss2email/config.py b/rss2email/config.py
+index 5c758bd..d73c8c2 100644
+--- a/rss2email/config.py
++++ b/rss2email/config.py
+@@ -186,7 +186,7 @@ CONFIG['DEFAULT'] = _collections.OrderedDict((
+         ('email-protocol', 'sendmail'),
+         # True: Use SMTP_SERVER to send mail.
+         # Sendmail (or compatible) configuration
+-        ('sendmail', '/usr/sbin/sendmail'),  # Path to sendmail (or compatible)
++        ('sendmail', '/run/current-system/sw/bin/sendmail'),  # Path to sendmail (or compatible)
```
Comment (Profpatsch) [inline pkgs/development/python-modules/rss2email-sendmail-default.patch]: This won’t work with non-nixos. It should be patched to either `/usr/bin/env sendmail` or to a package directly.

**Review by Profpatsch — CHANGES_REQUESTED** [review]

**Comment by jb55** [comment]
Profpatsch <notifications@github.com> writes:

> This won’t work with non-nixos. It should be patched to either `/usr/bin/env sendmail` or to a package directly.

ah right. maybe I should just leave it unpatched and let the user
configure it themselves? My idea here was to have something working out
of the box for nixos users...

**Comment by Profpatsch** [comment]
> ah right. maybe I should just leave it unpatched and let the user
> configure it themselves? My idea here was to have something working out
> of the box for nixos users...

Right, yes. But then just `sendmail` would be best I think. Maybe check if the program uses a call that resolves `PATH`.
Best of class would be

```
sendmail ? null, … }:
```

and use that or `"sendmail"` if `null`. Then the module can just add an option for that.

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -31828,6 +31828,28 @@ EOF
     };
   };
 
+  rss2email = buildPythonPackage rec {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: this is an application and not a Python library and thus doesn't belong in `python-packages.nix`.

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by jb55** [comment]
apparently I f'd this up bad so I'll take another pass at this later when I have time

**Comment by FRidh** [comment]
@jb55 it's not bad, it's just that adding a package to Nixpkgs can be tricky :)

**Pull request closed without merging**